### PR TITLE
feat: support loading eetq quantized model

### PR DESCRIPTION
**Pull request by thincal**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Was this discussed/approved via a Github issue or the discord / slack channel? Please add a link
      to it if that's the case.
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by thincal** [comment]
will revisit again after the #399 resolved.

**Comment by thincal** [comment]
@SidaZh is that possible help have a check this integration?

File: server/lorax_server/utils/weights.py [hunk 1]
```diff
@@ -226,6 +226,15 @@ def get_multi_weights_col(self, prefixes: List[Union[str, Tuple]], quantize: str
 
             bits, groupsize = self._get_gptq_params()
             weight = (qweight, qzeros, scales, g_idx, bits, groupsize, False)
+        elif quantize == "eetq":
+            try:
+                qweight = torch.cat(self.get_sharded_list("qweight", prefixes, dim=1), dim=1)
```
Comment (SidaZh) [inline server/lorax_server/utils/weights.py:231]: Is it necessary to merge the weight parameters of multiple cards here? The quantization of eetq includes two steps: quantization and cutlass relayout. If the tensor is sliced or concat, the layout will be destroyed and accuracy will be wrong.

Comment (SidaZh) [reply server/lorax_server/utils/weights.py:231]: There are two options: 1. Save and load using the same tp and pp strategies; 2. Load basic per-channel quantization parameters and relayout when initializing EETQLinear which requires some development. The following two interfaces will help:
```python
from EETQ import quant_weights, preprocess_weights

unprocessed_quantized_weight, processed_quantized_weight, scales = quant_weights(unquantized_weight, torch.int8, True)    # quantize and relayout
processed_quantized_weight = preprocess_weights(unprocessed_quantized_weight)   # relayout
```
